src/components/LazyLoadImage.jsx

```diff
@@ -17,8 +17,9 @@ class LazyLoadImage extends React.Component {
 			return null;
 		}
 
-		return () => {
-			this.props.afterLoad();
+		return e => {
+			this.props.onLoad(e);
+			this.props.afterLoad(e);
```

Comment:
What do you think about adding a comment here? Something along these lines:

```suggestion
			// We keep support for afterLoad for backwards compatibility,
			// but `onLoad` is the preferred prop.
			this.props.onLoad(e);
			this.props.afterLoad();
```

I would also not pass the event object to `afterLoad`, so we don't change the signature of that function.